<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMisc</class>
 <widget class="QDialog" name="DialogMisc">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>337</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>546</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tool Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/images/lupa.ico</normaloff>:/images/images/lupa.ico</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>511</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Select language</string>
   </property>
   <widget class="QComboBox" name="languageBox">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>21</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>English</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Czech</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>511</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Code template location</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>481</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLineEdit" name="templateLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="templateBtn">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>160</width>
     <height>26</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>511</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Default Workspace</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>481</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="lastUsedBtn">
       <property name="text">
        <string>Last Used</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="factoryBtn">
       <property name="text">
        <string>Factory Default</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="userWorkspaceBtn">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="userWorkspaceLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="selectWorkspaceBtn">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
